### PR TITLE
[DOC-9599] Document VIEWSYSTEMTABLE privilege's interaction with collecting statement diagnostic bundles

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-9599] Document VIEWSYSTEMTABLE privilege's interaction with collecting statement diagnostic bundles

Followup after https://github.com/cockroachdb/docs/pull/18204

### Previews
[src/current/v23.1/cockroach-statement-diag.md](https://deploy-preview-18294--cockroachdb-docs.netlify.app/docs/v23.1/cockroach-statement-diag.html)
[src/current/v23.2/cockroach-statement-diag.md](https://deploy-preview-18294--cockroachdb-docs.netlify.app/docs/v23.2/cockroach-statement-diag.html)

### CODE REVIEW

File: src/current/v23.1/cockroach-statement-diag.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Use the `cockroach statement-diag` [command]({% link {{ page.version.version }}/
 
 ## Required privileges
 
-Only members of the `admin` role can run `cockroach statement-diag`. By default, the `root` user belongs to the `admin` role.
+Only members of the `admin` role and users with the `VIEWSYSTEMTABLE` [system privilege]({% link v23.2/security-reference/authorization.md %}#privileges), which provides read-only access to system tables, can run `cockroach statement-diag`. By default, the `root` user belongs to the `admin` role.
```
Comment (rafiss) [inline src/current/v23.1/cockroach-statement-diag.md:12]: nit: this is linking to v23.2 docs; is that intentional?

**Review by rafiss — COMMENTED** [review]
nit: the PR title has VIEWSYSTEMDATA, but it should say VIEWSYSTEMTABLE

File: src/current/v23.1/cockroach-statement-diag.md (hunk 1 above)
Comment (mdlinville) [reply src/current/v23.1/cockroach-statement-diag.md]: Done

**Comment by mdlinville** [comment]
> nit: the PR title has VIEWSYSTEMDATA, but it should say VIEWSYSTEMTABLE

Fixed.

**Review by rafiss — APPROVED** [review]

**Review by kathancox — APPROVED** [review]
LGTM!

**Pull request merged**